docs/ui/workspaces.md

```diff
@@ -97,4 +97,36 @@ Select **Send** to initiate the invitation.
 
 To add a service account to a workspace, select the Service Accounts **+** icon. You can select from a list of existing service accounts configured for the organization. Select a Workspace Role for the service account. This will be the initial role for the service account within the workspace. A workspace Owner can change this role at any time. Select **Share** to finalize adding the service account.
 
-To delete a workspace collaborator or service account, select **Remove** from the menu on the right side of the user or service account information on this page.
\ No newline at end of file
+To delete a workspace collaborator or service account, select **Remove** from the menu on the right side of the user or service account information on this page.
+
+## Workspace transfer
+
+Workspace transfer enables you to move an existing workspace from a personal account to an organization, or between organizations.
```

Comment:
I'm assuming we're not eager to advertise that it also works from organization to personal?

Reply:
because that works the same